neo/sys/posix/platform_linux.cpp

```diff
@@ -56,21 +56,32 @@ Sys_EXEPath
 */
 const char* Sys_EXEPath()
 {
-	static char	buf[ 1024 ];
```

Comment:
Am Freitag, den 18.12.2015, 08:22 -0800 schrieb Daniel Gibson:
> > @@ -56,21 +56,32 @@ Sys_EXEPath
> >  */
> >  const char* Sys_EXEPath()
> >  {
> > -	static char	buf[ 1024 ];
> 
> why not just use `PATH_MAX`?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/RobertBeckebans/RBDOOM-3-BFG/pull/195/files#r48041
> 656

PATH_MAX is not friendly to porters...
https://www.gnu.org/software/hurd/community/gsoc/project_ideas/maxpath.
html

Usually there are functions that will allocate the memory for you.

--
tobi